cinema/views.py

```diff
@@ -56,3 +83,54 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    @staticmethod
+    def _params_to_str(qs):
+        return [str(str_id) for str_id in qs.split(",")]
+
+    def get_queryset(self):
```

Comment:
I guess this functionality can be improved in terms of compliance with SRP: for example, filtering may be extracted to a separate method.

Reply:
Come to think of it, it would probably be even better to do the same for `MovieSerializer`.